### DISCUSSION TITLE
Simulateur CFE

### DISCUSSION THREAD

**Discussion by mquandalle** [betagouv/mon-entreprise#2632] [Nouveaux simulateurs]
On a eu plusieurs demandes pour un simulateur CFE, la chose est cependant assez difficile a réaliser :

* Il y a plusieurs impôts en un dans l'avis d’impôt (CFE + taxe pour frais de CCI / TCMA)
* Le calcul des bases dépend de beaucoup de paramètres (déduction pour les artisans qui emploient des salariés, prise en compte des activités saisonnières, etc.)
* Les taux dépendent des collectivités locales et s'additionnent entre plusieurs échelons (commune, inter-co, syndicat de commune, taxe équipement, taxe GEMAPI), ces données ne sont pas facilement exploitable (la seule source que j'ai trouvé est l'API https://www.impots.gouv.fr/cll/zf1/accueil/flux.ex)
* Pour les entreprises qui n'ont pas de locaux, le calcul de la cotisation minimale n'est pas très clair

**Comment by hashar** [comment]
> Pour les entreprises qui n'ont pas de locaux, le calcul de la cotisation minimale n'est pas très clair

De mémoire la Cotisation Foncière des Entreprises est basée sur une valeur locative, on déclare je crois la surface des locaux.  Donc sans local 0 mètres carrés, ou une pièce pour les télétravailleurs (exemple 9 mètres carrés).  Mais les collectivités locales déterminent un plancher de valeur locative puis le taux est appliqué.

Exemple:

Surface: 9 mètres carrés
Valeur locative retenue: 1800 €
Taux: 33,3%
CFE: 600 €

On peut donc avoir une société avec une seule personne en télétravail payant autant de CFE qu'une société de 10 personnes dans un local de 150 mètres carrés.

**Comment by hashar** [comment]
Et il y a en OpenData le Fichier de recensement des éléments d'imposition à la fiscalité directe locale (REI). La source est la direction des impôts (DGFIP).

https://www.data.gouv.fr/fr/datasets/impots-locaux-fichier-de-recensement-des-elements-dimposition-a-la-fiscalite-directe-locale-rei-3/#resource-8a50a013-781a-4550-867a-b042eed91a07

Les champs sont décrits dans: https://www.impots.gouv.fr/portail/files/media/stats/trace_rei_complet_2018.xlsx on y retrouve des descriptions contenant le terme CFE:

* CFE - COMMUNE / BASES
* CFE - COMMUNE /TAUX NET
* CFE - SYNDICATS ET ORG. ASSIMILES / BASES
* CFE - SYNDICATS ET ORG. ASSIMILES /  TAUX NET
* CFE - INTERCOMMUNALITE / BASE / FP UNIQUE OU EN ZAE
* CFE - INTERCOMMUNALITE / TAUX NET / FP UNIQUE OU EN ZAE
* CFE - INTERCOMMUNALITE / BASE / FPE EN ZONE EOLIENNE
* CFE - INTERCOMMUNALITE / TAUX NET / FPE EN ZONE EOLIENNE

Etc. Ca à l'air très compliqué à gérer, à priori plus encore que le calcul du versement transport.

**Comment by mquandalle** [comment]
Ah oui effectivement @hasar, j'étais tombé sur ce fichier de taux il y a quelques temps mais je l'avais oublié. Bien vu !

Le problème pour la cotisation minimale, c'est qu'en plus du taux (que l'on a donc) il faut aussi connaître l'assiette minimale qui est défini au niveau des collectivités locales (avec un maximum légal en fonction du chiffre d'affaire) et dont on n'a pas les données à ma connaissance.

**Comment by hashar** [comment]
A noter que je n'ai pas d’intérêt personnel au calcul de la CFE, je pense que je peux simplement la demander au centre de fiscalité des entreprises dont je dépend. Si cette tache attend mon seul retour, elle peut effectivement être déclinée.

**Comment by laem** [comment]
J'ai tapé "simulateur CFE" dans Google, et je tombe sur cette issue. Aucun des liens avant n'a réussi à m'informer correctement 😅.
